ocaml/idl/datamodel.ml

```diff
@@ -5798,7 +5805,15 @@ let vif =
     ()
 
 let vif_metrics =
-  create_obj ~in_db:true ~in_product_since:rel_rio ~in_oss_since:oss_since_303 ~internal_deprecated_since:None ~persist:PersistNothing ~gen_constructor_destructor:false ~name:_vif_metrics ~descr:"The metrics associated with a virtual network device"
+  create_obj
+    ~internal_deprecated_since:(Some rel_tampa)
+    ~in_db:true
+    ~in_product_since:rel_rio
```

Comment:
Let's also remove this and add it to the new `~lifecycle` as a `Published` entry.